libfwupdplugin/fu-udev-device.c

```diff
@@ -10,9 +10,7 @@
 
 #include <fcntl.h>
 #include <string.h>
-#ifdef HAVE_ERRNO_H
-#include <sys/errno.h>
-#endif
+#include <errno.h>
```

Comment:
The reason for the weird ifdef is to allow windows cross compilation.

Reply:
I see, that (kinda?) makes sense!

Reply:
Fixed.